azure-pipelines.yml

```diff
@@ -0,0 +1,24 @@
+jobs:
+# Mac and Linux could potentially use the same template
+# except it isn't clear how to use a different build matrix
+# for each, so for now they are separate
+- template: buildscripts/azure/azure-macos.yml
+  parameters:
+    name: macOS
+    vmImage: xcode9-macos10.13
```

Comment:
Where did you find `xcode9-macos10.13` in the Microsoft docs? The [Python setup docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/python?view=vsts#example) only mention `'macOS-10.13'` for mac `vmImage` directive.

There's some mention of available software on [a github page](https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-Readme.md#xcode), but it still doesn't include the list of possible tags for `vmImage`.

Curious because I'm working on this for NumPy.

Reply:
I've noticed for example that incrementing the xcode version in the tag to i.e., `xcode10-macos10.13` fails only silently -- it will instead redirect to a windows build and run the build / tests without complaining. Looks like there are other ways to adjust xcode version though.

Reply:
You can find a list of the currently supported images here https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=vsts&tabs=yaml#use-a-microsoft-hosted-agent.

XCode10 has not been rolled out yet but it is coming.

Reply:
@tylerjereddy thanks for letting us know. I have a fix locally for the silent failure issue. I'll work on getting this fix rolled out.